src/download.test.ts

```diff
@@ -57,12 +57,6 @@ describe("downloadFile", () => {
 
     expect(url).toEqual(`${expectedUrl}/dir/test%3Fencoding${attachment}`);
   });
-
-  it("should download with query parameters being appended to the URL", async () => {
-    const url = await client.downloadFile(skylink, { query: { name: "test" } });
-
-    expect(url).toEqual(`${expectedUrl}?name=test&attachment=true`);
-  });
 });
```

Comment:
Recently we removed the `query` option for uploads. It had allowed devs to pass in custom parameters via query parameters, but a lot of those hadn't been implemented for large file uploads and if users were using `query` for that purpose it would have silently broke. I felt that it was a bit hacky anyway and decided to remove it from the API for downloads as well. Any custom parameters that are being used by devs should be supported in the API and documentation.